KeyPoint3/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Do not include .project in pull requests. This is local enviroment specific configuration.